File: python_code/LyricThief/TestClients/test_client.rb
#!/usr/bin/env ruby

require 'socket'

host = Socket.gethostname
port = 915

s = TCPSocket.new host, port

s.puts('{ "artist" : "MIA", "song" : "Paper Planes"}')

begin 
  while line = s.gets   # Read lines from the socket
    puts line.chop      # And print with platform line terminator
  end
rescue
end
s.close               # Close the socket when done

File: python_code/LyricThief/TestClients/test_client.py
#!/usr/bin/python

import socket               # Import socket module

s = socket.socket()         # Create a socket object
host = socket.gethostname() # Get local machine name
port = 915                  # Reserve a port for your service.

s.connect((host, port))

s.send('{ "artist" : "MIA", "song" : "Paper Planes"}')

print s.recv(10000)
s.close                     # Close the socket when done
